server/api/lib/trees/ramat_gan_tree_permit.js

```diff
@@ -0,0 +1,131 @@
+const proxy = require('./../proxy');
+const cheerio = require('cheerio');
+const TreePermit = require('../../model/tree_permit');
+const {
+	REGIONAL_OFFICE, START_DATE, PERMIT_NUMBER, APPROVER_TITLE, ACTION, 
+	END_DATE, LAST_DATE_TO_OBJECTION, TOTAL_TREES, REASON_DETAILED,
+	GUSH, HELKA, PLACE, STREET,
+	TREES_PER_PERMIT, PERMIT_ISSUE_DATE,
+} = require('../../model/tree_permit_constants');
+const { formatDate } = require('./utils');
+const Log = require('../log');
+
+const TREES_RAMAT_GAN_URL = `https://www.ramat-gan.muni.il/tashtiot/view/forest_commissioner/license/${new Date().getFullYear()}/`;
+const RGTreePermit = {
+	urls:[TREES_RAMAT_GAN_URL]
+};
+
+async function parseTreesHtml(url) {
+	const treesHtml = await proxy.get(url);
+	const dom = cheerio.load(treesHtml, {
+		decodeEntities: false
+	});
+	if (!dom) {
+		console.error('cheerio dom is null');
+	}
+	const keys = [];
+	const result = [];
+    
+    dom('.table_content_regular_wrap').find('TABLE').find('TR').each((row, elem) => {
+        if (row === 0) {
+            dom(elem).find('TH').each((idx, elem) => {
+                const key = dom(elem).text().trim();
+                keys.push(key);
+            });
+            return;
+        }
+        const treePermit = {};
+        dom(elem).find('TD,TH').each((idx, elem) => {
+            const value = dom(elem).text().trim();
+            if (value.length > 0) {
+                const key = keys[idx];
+                treePermit[key] = value;
+            }
+        });
+        if (Object.keys(treePermit).length > 0) {
+            result.push(treePermit);
+        }
+    });     
+    
+	Log.info(`number of ramat gan permits: ${result.length}`);
+	return result;
+}
+
+function processRawPermits(rawPermits) {
+	try {
+		const treePermits = rawPermits.map(raw => {
+            try {
+                const parts = raw['שם הרחוב'].split('\n'); // captures (כריתה), (העתקה)
+                const street = parts[0];
+                const action = parts.length > 1 ? parts[1] : 'כריתה';
+                const last_date_to_objection = parsePermitDates(raw['ניתן להגיש ערעור עד'])[0];
+                const treesPerPermit = raw['סוג העצים'] !== undefined ? parseTreesPerPermit(raw['סוג העצים']) : {};
+                const totalTrees = raw['סוג העצים'] !== undefined ? sum(Object.values(treesPerPermit)) : 0;
+                const dates = parsePermitDates(raw['מתאריך – עד תאריך']);
+                const permitNumber = `meirim-rg-${street}-${dates[0]}`;
+                
+                const attributes = {
+                    [REGIONAL_OFFICE]: 'רמת גן',
+                    [PLACE]: 'רמת גן',
+                    [APPROVER_TITLE]: 'פקיד יערות עירוני רמת גן',
+                    [PERMIT_NUMBER]: permitNumber,
+                    [STREET]: street,
+                    [ACTION]: action,
+                    [LAST_DATE_TO_OBJECTION]: last_date_to_objection,
+                    [REASON_DETAILED]: raw['סיבת הכריתה'],
```

Comment:
should be `REASON_SHORT`

Reply:
didnt get any of that, should talk

Reply:
fixed REASON_SHORT and person_request_name
not sure what you want to do about the url at this point

Reply:
did what i could to improve tree numbers 
its not perfect cause there is no end to the formats they put there, but its better

Reply:
did something with the url' hope its ok